### PR TITLE
LPS-168653 Replace liferay-ui:icon-menu with clay:dropdown-menu in asset_entries.jsp

### PR DESCRIPTION
**Pull request by BarbaraCabrera**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/asset_entries.jsp [hunk 1]
```diff
@@ -101,113 +100,24 @@ long[] groupIds = assetPublisherDisplayContext.getGroupIds();
...
 
-	</liferay-ui:icon-menu>
+<clay:dropdown-menu
+	additionalProps='<%=
+		HashMapBuilder.<String, Object>put(
+			"currentURL", currentURL
+		).build()
+	%>'
+	aria-label="<%= title %>"
+	cssClass="select-existing-selector"
```
Comment (victorg1991) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/asset_entries.jsp:114]: I think we can remove this class

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by ealonso** [comment]
Just started reviewing :)

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/10417

**Pull request closed without merging**